src/templates/home-page/index.tsx

```diff
@@ -1,123 +1,43 @@
-                key={_.label}
-                variant={index > 0 ? "outline" : undefined}
-                height="auto"
-                width="auto"
-                fontSize="lg"
-                px="40px"
-                py="20px"
-                isFullWidth
-                onClick={() => {
-                  window.open(_.url)
-                }}
-              >
-                {_.label}
-              </Button>
-            )
-          )}
-        </Stack>
-      </Box>
-      <Box as="section">
-        <H5 color="white" mt={32}>
-          {summary.title}
-        </H5>
-        <BodyLg
-          color="brand.100"
-          mt={6}
-          dangerouslySetInnerHTML={{ __html: summary.body }}
-        />
-        <Box position="relative">
-          {/* Divider that spans all 3 cards on large screen sizes */}
-          {!mediumBreakpoint && (
-            <Divider
-              position="absolute"
-              top="88px"
-              zIndex="1"
-              direction="horizontal"
-              borderColor="brand.300"
-            />
-          )}
-          <Stack
-            direction={{ base: "column", md: "row" }}
-            mt={8}
-            bg="blackAlpha.300"
-            divider={<StackDivider borderColor="brand.300" />}
-            border="1px solid"
-            borderColor="brand.300"
-          >
-            {summary.cards.map(
-              (
-                card: {
-                  title: string
-                  body: string
-                  icon: { alt: string; image: Omit<ImageProps, "alt"> }
-                },
-                i: number
-              ) => (
-                <SplashCard
-                  key={card.title}
-                  title={card.title}
-                  body={card.body}
-                  icon={{ ...card.icon.image, alt: card.icon.alt }}
-                  borderTop={
-                    mediumBreakpoint && i !== 0 ? "1px solid #9974FF" : 0
-                  }
-                  borderBottom={mediumBreakpoint ? "1px solid #9974FF" : 0}
-                  borderLeft={mediumBreakpoint || i === 0 ? 0 : undefined}
-                  borderRight={
-                    mediumBreakpoint || i === summary.cards.length - 1
-                      ? 0
-                      : undefined
-                  }
-                  isCardColumn={mediumBreakpoint}
-                />
-              )
-            )}
-          </Stack>
-        </Box>
-      </Box>
-    </Container>
+    <>
+      <Hero {...hero} />
+      <SectionTemplate {...stakerRole} />
+      <SectionTemplate {...lpRole} />
+      <SectionTemplate {...btcRole} />
+      <SectionTemplate {...tokenHolderRole} />
+      <TakeTheQuiz
+        topBgColor={tokenHolderRole.bgColor}
+        bottomBgColor={migrationInfo.bgColor}
+      />
+      <MigrationInfoSection {...migrationInfo} />
+      <HarnessThePower {...harnessThePower} />
+      <SectionTemplate {...activeCommunity} />
+      <CurrentProposals />
```

Comment:
Based on the Figma views the `current proposals` section should be a part of the `active community` section. https://www.figma.com/file/q8Aef9yTHUipGdnqzmh6UG/Website?node-id=1111%3A16916